docs/reference/mapping/types/search-as-you-type.asciidoc

```diff
@@ -162,15 +162,18 @@ GET my_index/_search
 The following parameters are accepted in a mapping for the `search_as_you_type`
 field and are specific to this field type
 
-[horizontal]
-
 `max_shingle_size`::
++
+--
+(Optional, integer)
+Largest shingle size to create. Valid values are `2` (inclusive) to `4`
+(inclusive). Defaults to `3`.
 
-    The largest shingle size to index the input with and create subfields for,
-    creating one subfield for each shingle size between 2 and
-    `max_shingle_size`. Accepts integer values between 2 and 4 inclusive. This
-    option defaults to 3.
+A subfield is created for each integer between `2` and this value. For example,
+a value of `3` creates two subfields: `my_field._2gram` and `my_field._3gram`
 
+More subfields enable more specific queries but increase index size.
```

Comment:
I have no idea whether the correct grammar is "more subfields _enable_" or "more subfields _enables_", but the UI copy is already shipped, so for consistency can we change on "more subfields _enables_"?

Reply:
I think it _should_ be plural ("_enable_") as "_subfields_" is also plural, but I can change this for consistency. Don't think it'll prevent anyone from understanding.

Reply:
I raised https://github.com/elastic/kibana/issues/60088 to get this fixed in UI copy. I'll update here once that's in.

Thanks again @cjcenizal!